app/src/main/java/com/flowfoundation/wallet/reactnative/bridge/BridgeModels.kt

```diff
@@ -152,6 +153,13 @@ class RNBridge {
         val rate: String
     )
 
+    data class SaveMnemonicResponse(
```

Comment:
We have promise already, why do we need this data model ? 
If there is error, throw it out, it will catch by rn end.